CHANGES.md

```diff
@@ -11,221 +11,212 @@
 
 **Full Changelog**: https://github.com/glue-viz/glue-jupyter/compare/v0.13.1...v0.14.0
 
-## [unreleased](https://github.com/glue-viz/glue-jupyter/compare/v0.13.1...HEAD)
-
-## [0.13.1](https://github.com/glue-viz/glue-jupyter/compare/v0.13.0...0.13.1) (2022-09-26)
+## [0.13.1](https://github.com/glue-viz/glue-jupyter/compare/v0.13.0...v0.13.1) - 2022-09-26
 
 ### What's Changed
 
 #### Bug Fixes
 
 - Fixed a bug causing the colour picker to enter an infinite loop upon
-- dragging in the widget. https://github.com/glue-viz/glue-jupyter/pull/312
-- 
-- Ensure `bqplot` `FRBImage` sends data as float32 to WebGL. https://github.com/glue-viz/glue-jupyter/pull/318, https://github.com/glue-viz/glue-jupyter/pull/319
-- 
+  dragging in the widget. in https://github.com/glue-viz/glue-jupyter/pull/312
 
-**Full Changelog**: https://github.com/glue-viz/glue-jupyter/compare/v0.13.0...v0.13.1
+- Ensure `bqplot` `FRBImage` sends data as float32 to WebGL. in https://github.com/glue-viz/glue-jupyter/pull/318, https://github.com/glue-viz/glue-jupyter/pull/319
 
-## [0.13.0](https://github.com/glue-viz/glue-jupyter/compare/v0.12.1...v0.13.0) (2022-08-26)
 
-### New Features
+## [0.13.0](https://github.com/glue-viz/glue-jupyter/compare/v0.12.1...v0.13.0) - 2022-08-26
 
-- Add button to profile viewer layers to toggle plotting as steps. https://github.com/glue-viz/glue-jupyter/pull/309
+### New Features
 
-**Full Changelog**: https://github.com/glue-viz/glue-jupyter/compare/v0.12.0...v0.13.0
+- Add button to profile viewer layers to toggle plotting as steps. in https://github.com/glue-viz/glue-jupyter/pull/309
 
-## [0.12.1](https://github.com/glue-viz/glue-jupyter/compare/v0.12.0...0.12.1) (2022-07-29)
+## [0.12.1](https://github.com/glue-viz/glue-jupyter/compare/v0.12.0...v0.12.1) - 2022-07-29
 
 - Fixed a bug that caused the image viewer zoom to be reset when adding
-- additional datasets. https://github.com/glue-viz/glue-jupyter/pull/316
+  additional datasets. in https://github.com/glue-viz/glue-jupyter/pull/316
```

Comment:
Is the addition of 'in' here deliberate?

Reply:
This is how the changelog-updater-action formats it, so I've adapted that style now for consistency. Although the tag comparisons are not consistent either (I like the compact links in the old releases better, but more importantly they were easier to produce with a line of perl :).